src/main/java/com/github/terma/jenkins/githubprcoveragestatus/CompareCoverageAction.java

```diff
@@ -195,12 +206,27 @@ private void publishStatusCheck(
         try {
             String text = message.forStatusCheck();
             List<GHPullRequestCommitDetail> commits = gitHubRepository.getPullRequest(prId).listCommits().asList();
+            String statusMessage;
+            boolean comparison;
+            if ("threshold".equalsIgnoreCase(comparisonMode)){
+                comparison = Percent.of(coverage) <= settingsRepository.getGreenThreshold();
+                statusMessage = message.forThresholdStatusCheck( settingsRepository.getGreenThreshold() );
+            } else if ("tolerance".equalsIgnoreCase(comparisonMode)){
+                float tolerance = settingsRepository.getTolerance();
+                float result = Math.abs(coverage - targetCoverage) * 100;
```

Comment:
This is incorrect, because it will result in coverage **increases** causing failures. I think this should be:

```java
float result = (targetCoverage - coverage) * 100;
```